### ISSUE TITLE
Remove factory

### ISSUE THREAD

**Issue by Dreamsorcerer** [aio-libs/aiocache#677]
factoyr.py can probably be removed. It's overly complicated and causes typing issues while seeming to provide no benefit at all.

I think it's probably preferable to simply import the caches the user wants explicitly:
```
from aiocache.backends.redis import RedisCache
```

**Comment by mikeoconnor0308** [comment]
Do you want to remove the support for CacheHandler / aliases as well?

**Comment by Dreamsorcerer** [comment]
Yes, I don't think the config adds much use and makes it impossible to fix #609.
